core/src/main/res/values/strings.xml

```diff
@@ -723,6 +723,7 @@
     <string name="search_powered_by">Search powered by %1$s</string>
 
     <string name="filter">Filter</string>
+    <string name="sort_order">Sort order</string>
```

Comment:
Is there already a string with the text "sort"? In that case, I would prefer using the existing one.